clusters/k8s-infra-dev-cluster-turnup/10-clusterconfiguration.tf

```diff
@@ -68,9 +69,21 @@ resource "google_container_cluster" "cluster" {
   name     = local.cluster_name
   location = local.cluster_location
 
-  provider           = google-beta
-  project            = data.google_project.project.id
-  min_master_version = data.google_container_engine_versions.cluster.latest_master_version
```

Comment:
Why are we deleting this?

Reply:
provider and project are still present, but I see no point in specifying a min_master_version - I want to use whatever GKE thinks is best and most stable (default).

Disagree?

Reply:
Sometimes GKE is behind, and we might need a newer version. GKE default is still `1.12.8-gke.10`. What if we need a feature in 1.13?